###############################################################################
# Parameter concerning the capturing of data
###############################################################################
pointCloudTopic:  /xtion/depth_registered/points
cameraFrame:      xtion_link
headPitchFrame:   HeadPitch_link
headYawFrame:     HeadYaw_link
torsoFrame:       torso
fixedFrame:       r_sole
footprintFrame:   base_footprint
minNumOfMeasurements: 3
bufferSize:       1000

###############################################################################
# Parameter concerning the detection of the ball
###############################################################################
minBallRadius:    0.074
maxBallRadius:    0.076
detectionRange:   1.5

###############################################################################
# Parameter concerning the optimization
###############################################################################
optimization: 

  HillClimbing_1_1:
      # Selects the optimization type.
      optimizationType:       HILL_CLIMBING
      
      # Selects whether the joint offsets should be calibrated or not.
      calibrateJointOffsets:  true 
      
      # Weight of the squared error between the estimated
      # marker position and the transformed measured positions.
      markerWeight:    1.0
      
      # Weight of the squared error of ground angle and ground distance.
      groundWeight:    1.0
      
      # Distance between footprint and ground.
      groundDistance:         0.02
      
      # Initial transformation:
      initial_x:              0.0
      initial_y:              0.0
      initial_z:              0.15
      initial_r:              0.0
      initial_p:              0.0
      initial_y:              0.0
     
  G2O_1_10000:
      # Selects the optimization type.
      optimizationType:       G2O
      
      # Selects whether the joint offsets should be calibrated or not.
      calibrateJointOffsets:  true
      
      # Weight of the squared error between the estimated
      # marker position and the transformed measured positions.
      markerWeight:     1.0
      
      # Weight of the squared error of ground angle and ground distance.
      groundWeight:     10000.0
      
      # Distance between footprint and ground.
      groundDistance:   0.02

